packages/ocamlformat-rpc/ocamlformat-rpc.0.18.0/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+synopsis: "Auto-formatter for OCaml code (RPC mode)"
+description:
+  "OCamlFormat is a tool to automatically format OCaml code in a uniform style. This package defines a RPC interface to OCamlFormat"
+maintainer: ["OCamlFormat Team <ocamlformat-dev@lists.ocaml.org>"]
+authors: ["Josh Berdine <jjb@fb.com>"]
+license: "MIT"
+homepage: "https://github.com/ocaml-ppx/ocamlformat"
+bug-reports: "https://github.com/ocaml-ppx/ocamlformat/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08" & < "4.13"}
+  "ocaml-version"
+  "ocamlformat-rpc-lib"
```

Comment:
is `ocamlformat-rpc-lib` always going to be released at the same time as `ocamlformat-rpc`? If yes, could `{= version}` be added?

Reply:
I guess they will be released at the same time but it shouldn't be an issue to have ocamlformat-rpc-lib 0.19.0 with ocamlformat-rpc 0.18.0 (or vice versa) in the future